docs/pages/architecture.md

```diff
@@ -9,18 +9,23 @@ permalink: architecture
 <a href="{{site.baseurl}}/assets/img/architecture/meshery-architecture.svg"><img src="{{site.baseurl}}/assets/img/architecture/meshery-architecture.svg" /></a>
 
 ## Clients
+
 <a href="{{site.baseurl}}/assets/img/architecture/meshery-architecture-clients.svg"><img src="{{site.baseurl}}/assets/img/architecture/meshery-architecture-clients.svg" /></a>
 
 ## Providers
+
 <a href="{{site.baseurl}}/assets/img/architecture/meshery-architecture-providers.svg"><img src="{{site.baseurl}}/assets/img/architecture/meshery-architecture-providers.svg" /></a>
 
-## Network Ports 
+## Network Ports
+
 Meshery uses the following list of network ports to interface with its various components:
 
-| Adapter       | Port          |
-| :------------ | :------------ |
-| Meshery REST API | 9081/tcp |
-{% assign adaptersSortedByPort = site.adapters | sort: 'port' -%}
+| Adapter                                        | Port             |
+| :--------------------------------------------- | :--------------- |
+| Meshery REST API                               | 9081/tcp      
+| Learn Layer5 Application                       | 10011 |
```

Comment:
@KarenEfereyan it would be good to append `/tcp` here.

Reply:
Will do. I wasnt sure if i was to add it.